.github/workflows/build-test.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Build/Test CI
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
```

Comment:
Pretty much the same as the Travis build, but test on both Node 14 and Node 16 (12 is active but no longer supported in ~1 month).

In my opinion, knowing that our package supports all active LTS versions of Node is worth the risk of 2x likelier to fail due to test instability. I'd rather identify and fix those tests anyways.